pytential/version.py

```diff
@@ -20,28 +20,12 @@
 THE SOFTWARE.
 """
 
-# {{{ find install- or run-time git revision
+from importlib import metadata
 
-import os
-if os.environ.get("AKPYTHON_EXEC_FROM_WITHIN_WITHIN_SETUP_PY") is not None:
-    # We're just being exec'd by setup.py. We can't import anything.
-    _git_rev = None
+from pytools import find_module_git_revision
 
-else:
-    import pytential._git_rev as _git_rev_mod
-    _git_rev = _git_rev_mod.GIT_REVISION
+VERSION_TEXT = metadata.version("pytential")
+VERSION = tuple([int(i) for i in VERSION_TEXT.split(".")])
```

Comment:
I noticed after I added this that the other projects have some regex for the version. Should this follow?

Reply:
This is OK, too.